### ISSUE TITLE
Market order

### ISSUE THREAD

**Issue by Ramblingjack** [xibalbas/paper_trading_view#6]
Sometimes the market order will not work and say invalid tp and sl even though it is not.

**Comment by alm0ra** [comment]
@Ramblingjack 
Please tell more information about your issue 
You can write an example that cause of this issue

**Comment by Ramblingjack** [comment]
Yes, once it happens to me again i will post an example.  so sometimes i wont be able to use market order, it will say invalid sl and tp. even though if i switch it to limit with exactly the same numbers, then it will work.

**Comment by alm0ra** [comment]
@Ramblingjack  i guess it because you choose a price near your enter price .
because in PTV i get last close price from trading view and when you want open new position some times this issue may be happen but please send me more information

**Comment by Ramblingjack** [comment]
I'll monitor it and provide more detailed information.

**Comment by Ramblingjack** [comment]
![image](https://user-images.githubusercontent.com/100504368/156328896-24039b7d-6c29-4c89-8db7-95306bd5c694.png)

**Comment by Ramblingjack** [comment]
if i choose limit and write in price 
![image](https://user-images.githubusercontent.com/100504368/156329004-b361daa1-e5c2-4251-b470-3038313b1244.png)

**Comment by Ramblingjack** [comment]
Although the trade did not open when i used the limit, even though it tagged the price on the next candle as well.

**Comment by alm0ra** [comment]
dear @Ramblingjack  
**first try to decrease speed of bar reply** do not use maximum speed .
second i guess it happen when we have a big change price . i mean a long candle but i will work on this problem .

**Comment by alm0ra** [comment]
@Ramblingjack if you like the project please give a star and tell others about this project . thanks

**Comment by Ramblingjack** [comment]
have already been doing the 2nd one, will do the first.
